### PR TITLE
boards: xtensa: m5stack_atom_lite: initial support

### PR DESCRIPTION
**Pull request by Trifunik**
Source code of board/xtensa/m5stickc_plus and  board/xtensa/m5stack_atoms3_lite were adapted for an inital support of the M5Stack ATOM Lite board.

### CODE REVIEW

File: boards/xtensa/m5stack_atom_lite/doc/img/m5stack_atom_lite.png
Comment (kartben) [inline boards/xtensa/m5stack_atom_lite/doc/img/m5stack_atom_lite.png:1]: Please look at using webp format for smaller file size, and use an illustration that's only the board's picture to be more consistent with other boards.

Comment (Trifunik) [reply boards/xtensa/m5stack_atom_lite/doc/img/m5stack_atom_lite.png:1]: done

Comment (Trifunik) [reply boards/xtensa/m5stack_atom_lite/doc/img/m5stack_atom_lite.png:1]: @kartben: Can you please check again? Thank you!

File: boards/m5stack/m5stack_atom_lite/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+   Hello World! m5stack_atom_lite
+
+Debugging
+---------
+
+M5Stack ATOM Lite debugging is not supported due to pinout limitations.
+
+Related Documents
+*****************
+- `M5Stack ATOM Lite docs <https://docs.m5stack.com/en/core/ATOM%20Lite>`
```
Comment (kartben) [inline boards/m5stack/m5stack_atom_lite/doc/index.rst:132]: please check this link https://builds.zephyrproject.io/zephyr/pr/68190/docs/boards/m5stack/m5stack_atom_lite/doc/index.html

I guess it shoudl be:

```suggestion
- `M5Stack ATOM Lite docs <https://docs.m5stack.com/en/core/ATOM%20Lite>`_
```

**Review by kartben — CHANGES_REQUESTED** [review]
Thanks! Looks good to me except for a small documentation typo. Will add a few Espressif folks to help with reviewing this

File: boards/m5stack/m5stack_atom_lite/doc/index.rst (hunk 1 above)
Comment (Trifunik) [reply boards/m5stack/m5stack_atom_lite/doc/index.rst]: Thank you for the quick response!
Done!

**Review by kartben — APPROVED** [review]

File: boards/m5stack/m5stack_atom_lite/m5stack_atom_lite_esp32_appcpu.dts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		};
+
+		/* Reserve 1024kB for the application in slot 1 */
+		slot1_partition: partition@110000 {
+			label = "image-1";
+			reg = <0x00110000 0x00100000>;
+		};
+
+		/* Reserve 256kB for the scratch partition */
+		scratch_partition: partition@210000 {
```
Comment (sylvioalves) [inline boards/m5stack/m5stack_atom_lite/m5stack_atom_lite_esp32_appcpu.dts:64]: @kartben @nordicjm Should we start requesting to not having scratch area at all? Or better keep this until mcuboot has updated it accordingly?

**Review by sylvioalves — APPROVED** [review]

File: boards/m5stack/m5stack_atom_lite/m5stack_atom_lite_esp32_appcpu.dts (hunk 2 above)
Comment (nordicjm) [reply boards/m5stack/m5stack_atom_lite/m5stack_atom_lite_esp32_appcpu.dts:64]: Up to you as the vendor but scratch hasn't been needed in MCUboot for years, see https://github.com/zephyrproject-rtos/zephyr/commit/9a842588fb05ee16c8e0a5f7c3217bdfb561c1d4

**Comment by Trifunik** [comment]
Hello, many thanks for the quick review!
Unfortunately, after the last push a twister test failed. Does anybody know, 
how I can connect the administrator if twister writes: “Executing the custom container implementation failed. Please contact your self hosted runner administrator.” ?
I also don’t understand this error. Because the failed test is for qemu_x86_64 and I have not changed anything in this part of the source code.

**Comment by kartben** [comment]
> @nordicjm: Thank you for rerunning the tests! Unfortunately, it failed again. How severe is this failure? Because as I understand, this pull request will not be merged until all checks are successful. But the errors are happening on the qemu_x86_64 architecture and this has nothing to do with my esp32 based board.
> 
> Many thanks in advance! Nikola

I re-ran the job and it passed

**Pull request merged**